sw/device/rom_exts/rom_ext_start.S

```diff
@@ -183,50 +183,43 @@ _rom_ext_start_boot:
    */
 
   /**
-   * Initialize the `.data` section.
-   *
-   * `t0` is the start address of `.data` (in RAM).
-   * `t1` is the end address of `.data` (in RAM).
-   * `t2` is the start address of `.data` (in ROM).
-   * `t3` is a scratch register for the copy.
+   * Initialize the `.data` section in RAM from ROM.
    */
-  la   t0, _data_start
-  la   t1, _data_end
-  la   t2, _data_init_start
-  bgeu t0, t1, data_copy_loop_end
-data_copy_loop:
-  // TODO: Unroll this loop
-  lw   t3, 0x0(t2)
-  sw   t3, 0x0(t0)
-  addi t0, t0, 0x4
-  addi t2, t2, 0x4
-  bltu t0, t1, data_copy_loop
-data_copy_loop_end:
+  la   a0, _data_start
+  la   a1, _data_end
+  la   a2, _data_init_start
+  .extern crt_section_copy
```

Comment:
In other files the `.extern` directives are at the top? Worth being consistent.

Reply:
The mask ROM code has `.extern` directives close to their uses. I'm not certain which we should pick. Maybe something to make consistent in a different PR?